### ISSUE TITLE
Why we have self.Q = tf.reduce_sum(tf.multiply(self.output, self.actions_), axis=1) ?

### ISSUE THREAD

**Issue by Meur-sault** [simoninithomas/Deep_reinforcement_learning_Course#61]
Hi Thomas,

(Since this issue got resolved without any proper answer, I'm submitting it again.)
I don't understand that why we are doing tf.reduce_sum and multiple the network output to action.

In [7]: ....... 
# Q is our predicted Q value
self.Q = tf.reduce_sum(tf.multiply(self.output, self.actions_), axis=1)

Why aren't we considering self.output as predicted Q value.
